RecoCTPPS/TotemRPLocal/plugins/TotemRPUVPatternFinder.cc

```diff
@@ -188,10 +188,11 @@ void TotemRPUVPatternFinder::produce(edm::Event& event, const edm::EventSetup& e
 
   for (auto &ids : *input)
   {
-    unsigned int detId = TotemRPDetId::rawToDecId(ids.detId());
-    unsigned int rpId = TotemRPDetId::rpOfDet(detId);
-    unsigned int plane = detId % 10;
-    bool uDir = TotemRPDetId::isStripsCoordinateUDirection(detId);
+    TotemRPDetId detId(ids.detId());
+    unsigned int plane = detId.plane();
+    bool uDir = detId.isStripsCoordinateUDirection();
+
+    TotemRPDetId rpId = detId.getRPId();
```

Comment:
Hi @jan-kaspar - i find this pretty confusing - maybe we have simply overloaded the meaning of RP? This is asking a TotemRPDetId object for its RPId (which is implemented in CTPPSDetId..) - can you clarify?